app/base/utils.py

```diff
@@ -52,60 +23,26 @@ def generate_img_dir_name():
     return dir_name
 
 
-def property_image_handler(temp_dir, image_file_list):
-    """
-    Save the images. First a new directory is created where the images are saved. The process_images() is called and
-    runs in the background. Then the directory name and image filenames are returned in one list.
-    """
-
-    from app.tasks import process_images
-
-    generated_dirname = generate_img_dir_name()
-    images_list = [generated_dirname]
-    for filename in image_file_list:
-        images_list.append(filename)
-        process_images.delay(filename, temp_dir, generated_dirname)
-    return images_list
-
-
-def update_property_images(temp_dir, image_file_list, prop_id):
-    """
-    Updates the images. First the old directory is deleted and then a new one is created
-    where the images are saved. The process_images() is called and runs in the background.
-    Then the directory name and image filenames are returned in one list.
-    """
-    from app.tasks import process_images, delete_img_objs
-
-    prop_to_update = Property.query.get(prop_id)
-    prop_images = json.loads(prop_to_update.photos)
-    path_to_del = s3_prop_image_dir + prop_to_update.image_folder
-    delete_img_objs.delay(bucket, path_to_del, prop_images)
-
-    generated_dir_name = generate_img_dir_name()
-
-    images_list = [generated_dir_name]
-    for filename in image_file_list:
-        images_list.append(filename)
-        process_images.delay(filename, temp_dir, generated_dir_name)
-    return images_list
-
-
-def save_profile_picture(image_file):
+def save_to_redis(image_file_list):
     """
-    Handle the uploaded image file. The file is renamed to a random hex, then the image object is process by PIL
-    image library then saved.
+    Saves the images to redis by converting it to bytes. Returns the keys of each imagein a list.
     """
-    from app.tasks import update_profile_image
+    redis_img_keys = []
+    for file in image_file_list:
+        rand_str = str(uuid.uuid4())[:8]  # generate a random string to set as key for the object in redis
+        redis_img_keys.append(rand_str)
+        # convert file to bytes
+        img_to_bytes = np.array(np.frombuffer(file.read(), np.uint8)).tobytes()
+        redis_client.set(rand_str, img_to_bytes)
```

Comment:
I would really  like to  give this name rand_str something readable and meaningful